ui/components/component-library/banner-base/banner-base.js

```diff
@@ -91,15 +91,15 @@ BannerBase.propTypes = {
   /**
    * Additional props to pass to the `Text` component used for the `title` text
    */
-  titleProps: PropTypes.shape(Text.PropTypes),
+  titleProps: PropTypes.object,
```

Comment:
Unfortunately using `Text.propTypes` breaks so need to update proptypes to use generic object while during our migration process

Reply:
Is there a task captured for returning to PropTypes?

Reply:
Not a particular issues addressing the propTypes but is it possible to use types from a TypeScript component with PropTypes. The issue that would fix this in future ins the migration of the component to TypeScript